docs/monitor/configure-alarms.md

```diff
@@ -31,7 +31,7 @@ For example, to edit the `cpu.conf` health configuration file, run:
 sudo ./edit-config health.d/cpu.conf
 ```
 
-Each health configuration file contains one or more health _entities_, which always begin with `alarm:` or `template:`.
+Each health configuration file contains one or more health _entities_, which always begin with `alert:` or `template:`.
```

Comment:
Hi Samantha!

On this particular place, we can't change `alarm` to `alert` since that is a keyword present in health configuration files and is required by the agent when it reads configurations. (And if we decide to change the keyword, we'll need to do some work in agent).

Thanks!